src/Components/Facility/DischargeModal.tsx

```diff
@@ -130,6 +137,18 @@ const DischargeModal = ({
       return;
     }
 
+    const dischargeDetails = {
+      ...preDischargeForm,
+      discharge: value,
+      discharge_date: moment(preDischargeForm.discharge_date).toISOString(true),
```

Comment:
@sachdevavaibhav

- [ ] Merge the latest develop (because we moment has been uninstalled)
- [ ] Discard the `package-lock.json` diff
- [ ] replace `moment` with `dayjs`

Reply:
@rithviknishad Kindly review